tower-http/src/timeout/mod.rs

```diff
@@ -0,0 +1,132 @@
+//! Middleware that adds timeouts to request or response bodies.
+//!
+//! Note these middleware differ from [`tower::timeout::Timeout`] which only
+//! adds a timeout to the response future and doesn't consider request bodies.
```

Comment:
What does this mean for `ResponseBodyTimeout`? Is it equivalent to the tower middleware?